src/Symfony/Component/Validator/Constraints/EmailValidator.php

```diff
@@ -133,6 +133,8 @@ public function validate($value, Constraint $constraint)
 
         // Check for host DNS resource records
         if ($constraint->checkMX) {
+            @trigger_error(sprintf('The %s::$checkMX property is deprecated since Symfony 4.2.', Email::class), E_USER_DEPRECATED);
```

Comment:
basically this triggers a second deprecation..  we didnt do it for Url either

Reply:
I added it because that's what we've done for strict, but I agree with you that it makes things more verbose. Let's remove that.